include/librepr/ctvi/ctvi.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <string_view>
+#include <array>
+#include <utility>
+
+namespace librepr::ctvi {
+namespace detail {
+template <std::size_t... Idx>
+consteval auto sv_to_array(std::string_view str, std::index_sequence<Idx...>) noexcept
+    -> std::array<char, (sizeof...(Idx) + 1)> {
+  return {{str[Idx]..., '\0'}};
+}
+
+template <auto V>
+[[nodiscard]] constexpr auto get_ctvi() noexcept {
```

Comment:
Gotta make all the ``__PRETTY_FUNCTION__`` related stuff to compiler independent